brands/shop/kitchen.json

```diff
@@ -21,6 +21,8 @@
     "count": 58,
     "tags": {
       "brand": "Mobalpa",
+      "brand:wikidata": "Q3317571",
```

Comment:
from the warnings on the `official_website` claim, it looks like there may be two entries for Mobalpa:
Q3117411 and Q3317571

Lets figure out which one is better and sort it out on Wikidata before we merge here..

Reply:
Q3117411 is the parent company that bought Mobalpa, it should not have `official_website` set to `http://mobalpa.com`